plugins/plugin-site/src/components/ThankAContributorNote.jsx

```diff
@@ -0,0 +1,150 @@
+/* eslint-disable react/jsx-one-expression-per-line */
+
+import React, {useEffect, useState} from 'react';
+import axios from 'axios';
+import dayjs from 'dayjs';
+import {Box, Stack, useTheme} from '@mui/material';
+import Papa from 'papaparse';
+import {useMediaQuery} from '../hooks/useMediaQuery';
+
+function ThankAContributorNote() {
+    const theme = useTheme();
+    const isMobile = useMediaQuery('man-width: 768px)');
+    const isDesktop = useMediaQuery('(min-width: 1024px)');
+
+    const [thankYou, setThankYou] = useState([]);
+
+    useEffect(() => {
+        axios
+            .get(
+                'https://raw.githubusercontent.com/jenkins-infra/jenkins-contribution-stats/main/data/honored_contributor.csv',
+                {responseType: 'text'}
+            )
+            .then((response) => {
+                setThankYou(Papa.parse(response.data)?.data[1]);
+            });
+
+        const interval = setInterval(() => {
```

Comment:
This is to make sure the contributor is refreshed after 1 hour of having the page open? I don't think that's needed.

Reply:
It is more a failsafe mechanism